dropbox-sdk-dotnet/Dropbox.Api/Dropbox.Api.csproj

```diff
@@ -1,48 +1,23 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>AppProperties</AppDesignerFolder>
-    <RootNamespace>Dropbox.Api</RootNamespace>
-    <AssemblyName>Dropbox.Api</AssemblyName>
-    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
-    <BaseIntermediateOutputPath>obj\</BaseIntermediateOutputPath>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <TargetFramework>netstandard2.0</TargetFramework>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-  </PropertyGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net45' ">
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="System.Net.Http" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Xml" />
-    <Reference Include="Newtonsoft.Json">
-      <HintPath>..\packages\Newtonsoft.Json.7.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
-    </Reference>
-  </ItemGroup>  
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net45' ">
-    <None Include="packages.config" />
-  </ItemGroup>
+    <Title>Dropbox.Api</Title>
+    <Description>Official Dropbox .Net v2 SDK</Description>
+    <Company>Dropbox Inc.</Company>
+    <Product>Dropbox.Api</Product>
+    <Copyright>Copyright © Dropbox Inc</Copyright>
```

Comment:
I believe we want a year with the copyright per the discussion I had with the open source team. Not sure how this field gets put into the actual distributable though. But just add 2020 for the year assuming it is needed here